client/app/components/ListSlider.js

```diff
@@ -21,7 +21,7 @@ const ListSlider = ({ currentIndex, itemList, renderItem }) => {
 
 ListSlider.propTypes = {
 	currentIndex: PropTypes.number.isRequired,
-	itemList: PropTypes.array.isRequired,
+	itemList: PropTypes.arrayOf(PropTypes.any).isRequired,
```

Comment:
Instead of using this, I'd rather you use `"react/forbid-prop-types": 0` in the eslint JSON

Reply:
Hm. I want forbidden types to be in place to check for errors in the arrays we are passing in.
Once we know what the itemList exactly contains, I'll change this portion.

Reply:
It's not just for itemList, it's also for navigation.  If you want to leave it in, find out what is in it.  For navigation, it might be tougher to figure out exactly what is in there since it's an object, not an array.  One way to do it is to disable eslint each time we pass in `navigation`.  If you can figure this out, that would be awesome.  If not, I don't think we should leave `PropTypes.any` in the code.  (Specifying it's an object > Instance of any type)